### PR TITLE
Docs: Update README with deprecation notice

### PR DESCRIPTION
**Pull request by nzakas**
This represents what should be the final README for this package as we deprecate it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# This project has been moved to the TypeScript ESLint organization -> https://typescript-eslint.io
+# Deprecated: TypeScript ESLint Parser
 
-The `typescript-eslint-parser` npm package will not receive any future updates, instead please follow along on the new monorepo, where the project will be released under the name `@typescript-eslint/parser`.
+**Important:** This repository is no longer maintained and `typescript-eslint-parser` will not receive any future updates. There is an actively maintained fork of this project available at https://typescript-eslint.io and published on npm as `@typescript-eslint-parser`.
```
Comment (platinumazure) [inline README.md]: Per above, here's a suggested change to make this easier to tweak:

```suggestion
**Important:** This repository is no longer maintained and `typescript-eslint-parser` will not receive any future updates. There is an actively maintained fork of this project available at https://typescript-eslint.io and published on npm as `@typescript-eslint/parser`.
```

**Review by platinumazure — CHANGES_REQUESTED** [review]
LGTM aside from the new package name. Thanks!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nzakas) [reply README.md]: Oops, thanks!

Comment (nzakas) [reply README.md]: Thank you, sir.

**Review by platinumazure — APPROVED** [review]
LGTM, thanks!

**Review by kaicataldo — APPROVED** [review]
LGTM!

**Review by JamesHenry — APPROVED** [review]

**Comment by JamesHenry** [comment]
Thanks!

**Review by armano2 — APPROVED** [review]

**Pull request merged**